packages/core/stories/card/card.doc.stories.mdx

```diff
@@ -26,38 +26,70 @@ This default example is intended to show the way the Card component’s content
   <Story id="core-card--default" />
 </Canvas>
 
-## Interactable
+## Interactable Card
```

Comment:
We should have content review this to make sure it's not confusing having two Interatable Card headers